src/api-reference/travel/itinerary/trip/trip-resource.markdown

```diff
@@ -360,49 +367,49 @@ Where `access_token` is the OAuth 2.0 access token of the user whose itinerary i
 
 application/xml
 
-## Response
+## <a name="gettd-schema"></a>Get Trip Details Response Schema
```

Comment:
This is getts-schema in the TOC. Which is correct?

Reply:
Actually, I didn't add a schema entry for this heading. getts-schema was the previous one.
 Fixed.